helpers/providers/gemini.ts

```diff
@@ -0,0 +1,73 @@
+import ciInfo from "ci-info";
+import prompts from "prompts";
+import { ModelConfigParams } from ".";
+import { questionHandlers, toChoice } from "../../questions";
+
+// TODO: get embedding vector dimensions from the google sdk (currently not supported)
+// Gemini has the same name for the model and the embedding model
+// Get dimensions data from: https://cloud.google.com/vertex-ai/generative-ai/docs/learn/models#gemini-models
```

Comment:
seems the embedding models are called: textembedding-gecko and textembedding-gecko-multilingual
see https://cloud.google.com/vertex-ai/generative-ai/docs/embeddings/get-text-embeddings

Reply:
the first one is using `768` dimensions - multilingual I didn't find the dimensions - either test or remove

Reply:
yes, please test at least `textembedding-gecko`